docs/pages/server-access/guides/vscode.mdx

```diff
@@ -100,51 +100,45 @@ When you see this error, re-run `tsh login` to refresh your local certificate.
 
 ## Step 2/3. Configure Visual Studio Code
 
-<Admonition
-  type="warning"
-  title="Warning"
->
-  Due to [a VS Code bug](https://github.com/microsoft/vscode-remote-release/issues/5562),
-  you can not use Remote - SSH UI to add SSH hosts. It will improperly rewrite the quoting in
-  `tsh config`'s generated configuration.
-  If your SSH configuration becomes corrupted, re-run `tsh config`.
-</Admonition>
-
-Install the [Remote - SSH extension][remote-ssh] in your local VS Code
-instance. A new "Remote Explorer" sidebar entry should appear.
+Install the [Remote - SSH extension][remote-ssh] in your local VS Code instance. 
```

Comment:
@ptgott This link doesn't look right, any idea where it should be going to?

Reply:
It goes to [this page](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-ssh), which I think is correct (the link reference isn't showing up in the diff because it wasn't changed).

Reply:
Got it; I didn't realise you could add static references within a doc for that. Thanks!